vhost-device-vsock/src/vsock_conn.rs

```diff
@@ -66,6 +69,7 @@ impl<S: AsRawFd + ReadVolatile + Write + WriteVolatile> VsockConnection<S> {
         guest_port: u32,
         epoll_fd: RawFd,
         tx_buffer_size: u32,
+        write_ok: bool,
```

Comment:
Instead of adding a new parameter, can we use `stream` to get this information?

Reply:
Good point. I have now introduced a IsUnixStream trait for StreamType and used it instead of passing 'write_ok'.